data/src/main/java/org/oppia/android/data/backends/gae/api/PlatformParameterService.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.android.data.backends.gae.api
+
+import org.oppia.android.data.backends.gae.model.GaePlatformParameters
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/** Service that provides access to platform parameter endpoint. */
+interface PlatformParameterService {
+
+// TODO("Change the url to point to the correct endpoint when the backend is ready")
```

Comment:
Is there no backend for this yet? I had thought the backend was done in a previous project.

Reply:
@BenHenning and @jcqli currently we have an endpoint in Oppia for platform parameters but making any changes to it was discussed to be put in the Future Works in the Proposal. Thats why I have mentioned this as TODO becuase we have not confirmed that whether we will be sending version number to the backend or the names of paramaters to be returnined in the response. //cc: @vinitamurthi

Reply:
Ack, make an issue for it an include the issue # here then.

Reply:
Done

Reply:
This actually seems a bit concerning to me. We shouldn't be setting up the model layer for an endpoint format that doesn't exist. Either these API endpoint representations should match the current backend, or the current backend should be updated to fit these. Otherwise, the system isn't going to work. 'Future work' should be optional, not required, pieces of work beyond the defined requirements for the system to work within spec.